src/OperationKeyboard/controller.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+import { EventDispatcher } from '../eventDispatcher.js';
+import viewTemplate from './view.hbs';
+import stringToDom from 'string-to-dom';
+
+export const OperationKeyboardEvents = {
+    RESET_ON_NEXT_INPUT: 'resetOnNextInput',
+    GET_OPERATION: 'getOperation'
+};
+
+export class OperationKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindOpsEvents();
+    }
+    /**
+     * Binds event listeners to operation buttons.
+     */
+    _bindOpsEvents() {
+        let viewString = viewTemplate();
+        this.view = stringToDom(viewString);
+        let operationButtons = this.view.querySelectorAll('.operation');
+        for (let i = 0; i < operationButtons.length; i++) {
+            operationButtons[i].addEventListener("click", (event) => {
+                this.trigger(OperationKeyboardEvents.RESET_ON_NEXT_INPUT)
+                let operation = event.target.textContent;
+                this.trigger(OperationKeyboardEvents.GET_OPERATION, operation);
```

Comment:
This is a bit confusing name for event, since it is not clear what it represents. You could use something like:
```
OperationKeyboardEvents.OPERATION_SELECTED
```